docs/technical_reference/flowsheets/BSM2.rst

```diff
@@ -52,6 +61,131 @@ Documentation for each of the property models can be found here:
     * `ASM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ASM1.html>`_
     * `ADM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ADM1.html>`_
 
+Degrees of Freedom
+------------------
+The following variables must be specified by the user to run the BSM2 flowsheet:
+    * feed water conditions (flow, temperature, pressure, component concentrations, and alkalinity)
+    * volume of activated sludge reactors
+    * component injection rates for aerobic reactors
+    * split fraction(s) for the recycle loop after the activated sludge reactors
+    * secondary clarifier surface area and split fraction(s)
+    * primary clarifier split fraction(s)
+    * split fraction(s) for the separator following the secondary clarifier
+    * pressure changer outlet pressure (feeds into the activated sludge process)
+    * anaerobic digester liquid volume, vapor volume, and liquid outlet temperature
+    * dewatering unit hydraulic retention time and specific energy consumption
+    * thickener hydraulic retention time and diameter
+
+Flowsheet Specifications
```

Comment:
can we add any information from where our specifications came from?

Reply:
I've added the references used for the feed water concentrations.